cmd/download.go

```diff
@@ -187,8 +190,8 @@ func saveToImagesFile(image, imageMapping string, aiImages []string, aiImagesFil
 
 func download(folder, release, url string,
 	aiImages, additionalImages []string,
-	rmStale, generateImageSet, skipImageSet bool,
-	extraArgs []string) {
+	rmStale, generateImageSet, duProfile bool,
+	skipImageSet bool, extraArgs []string) {
```

Comment:
nit: Why move the skipImageSet bool to a new line, instead of just inserting your new bool?